src/main/java/com/example/demo/controller/MailSenderController.java

```diff
@@ -0,0 +1,44 @@
+package com.example.demo.controller;
+
+import com.example.demo.model.Message;
+import com.example.demo.repo.JsonRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController()
+@RequestMapping("/message")
+public class MailSenderController {
+
+    @Autowired
+    private JsonRepo jsonRepo;
+
+    @GetMapping()
+    public List<Message> getMessages() {
+        List<Message> sender = jsonRepo.loadFromJsonToList();
+        return sender;
+    }
+
+    @GetMapping("/{id}")
+    public Message getMessage(@PathVariable int id) {
+        Message message = jsonRepo.findById((long) id);
+        return message;
+    }
+
+    @PostMapping("/new")
```

Comment:
We should avoid using verbs in url name. Url is the name of the resource and http method(a verb) which tells what to with a resource

Reply:
Done